test/middleware.ts

```diff
@@ -0,0 +1,264 @@
+var assert = require('assert')
+var http = require('http')
+var morgan = require('morgan')
+var request = require('supertest')
+var split = require('split')
+var kayvee = require('../lib/kayvee')
+var kv_middleware = require('../lib/middleware')
+
+describe('middleware', function () {
+  var expected;
+  it('should pass default fields', function (done) {
+    var cb = afterTest(2, function (err, res, line) {
```

Comment:
Why is this `2` here? My understanding is that this should be the # of logs, but it looks like we only make one request to the server?